Sources/FBLPromises/include/FBLPromise+Testing.h

```diff
@@ -46,6 +46,11 @@ FOUNDATION_EXTERN BOOL FBLWaitForPromisesWithTimeout(NSTimeInterval timeout) NS_
  */
 @property(nonatomic, readonly, nullable) NSMutableSet *pendingObjects NS_REFINED_FOR_SWIFT;
 
+/**
+ Add an object to the set of pending objects to keep strongly while the promise is pending.
+ */
+- (void)addToPendingObjects:(id)object NS_REFINED_FOR_SWIFT;
```

Comment:
Doesn't seem we need that readonly property above anymore, this method should be sufficient.
Can you remove that please?

Reply:
Sure. The only reason why it was needed is that it's used in a bunch of places in tests to confirm that the pending objects are empty.